pkg/cluster/update_master.go

```diff
@@ -75,7 +75,7 @@ func (u *simpleUpgrader) UpdateMasterAgentPool(ctx context.Context, cs *api.Open
 		}
 
 		u.log.Infof("reimaging %s", computerName)
-		err = u.vmc.Reimage(ctx, cs.Properties.AzProfile.ResourceGroup, ssName, *vm.InstanceID, nil)
+		_, err = u.vmc.ReimageAll(ctx, cs.Properties.AzProfile.ResourceGroup, ssName, *vm.InstanceID)
```

Comment:
@charlesakalugwu you need to wait on the future returned by ReimageAll.  Write a shim function like this one: https://github.com/openshift/openshift-azure/blob/master/pkg/util/azureclient/compute_addons.go#L120-L128

Reply:
@charlesakalugwu does this wipe the etcd disk as well?  We definitely do not want to do that.

Reply:
@jim-minter i just discovered that ReimageAll wipes all managed disks of a VM. This is definitely not what we want for the masters.